CHANGES.rst

```diff
@@ -68,6 +68,9 @@ Other changes
 - provided method to get breadcrumb length to prevent ZMI errors
   (`#533 <https://github.com/zopefoundation/Zope/issues/533>`_)
 
+- add zodbupdate_rename_dict to move webdav.LockItem to OFS.LockItem
+  (https://github.com/plone/Products.CMFPlone/issues/2800>)
```

Comment:
The formatting is broken, suggestion:
(`Products.CMFPlone#2800 <https://github.com/plone/Products.CMFPlone/issues/2800>`_)

Reply:
done